bash/code_to_gast/bash_main.js

```diff
@@ -0,0 +1,14 @@
+const bash_router = require("./bash_router.js")
+const parse = require("bash-parser");
+
+const bash_input = process.argv[2]
+let input_ast = ""
+try {
+    input_ast = parse(bash_input);
+} catch(err) {
+    return "Error: code could not compile"
+}
+
+const gast = bash_router.node_to_gast(input_ast)
+const json_string = JSON.stringify(gast)
+console.log(json_string)
```

Comment:
Is this how the result is sent back to the caller?

Reply:
Yes